### ISSUE TITLE
Can't open any projects on hosted on m1 in expo go on real device or simulator

### ISSUE THREAD

**Issue by dereknelson** [expo/expo#12376]
### Summary

If the exp server is hosted on an m1 device, the expo go app hangs at "new update available, downloading". fails to open tunnel as well. ~this began this morning, I'm assuming it's related to SDK 41 beta.~
I've reinstalled the app on both simulator and real device, still hangs.

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

managed

### What platform(s) does this occur on?

iOS

### SDK Version (managed workflow only)

40

### Environment

```  
Expo CLI 4.3.2 environment info:
    System:
      OS: macOS 11.1
      Shell: 5.8 - /bin/zsh
    Binaries:
      Node: 14.15.1 - ~/.nvm/versions/node/v14.15.1/bin/node
      Yarn: 1.22.10 - /usr/local/bin/yarn
      npm: 6.14.8 - ~/.nvm/versions/node/v14.15.1/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    Managers:
      CocoaPods: 1.10.1 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: iOS 14.4, DriverKit 20.2, macOS 11.1, tvOS 14.3, watchOS 7.2
    IDEs:
      Android Studio: 4.1 AI-201.8743.12.41.6953283
      Xcode: 12.4/12D4e - /usr/bin/xcodebuild
    npmPackages:
      expo: ~40.0.0 => 40.0.1 
      react: 16.13.1 => 16.13.1 
      react-dom: 16.13.1 => 16.13.1 
      react-native: https://github.com/expo/react-native/archive/sdk-40.0.1.tar.gz => 0.63.2 
      react-native-web: ~0.13.12 => 0.13.18 
    Expo Workflow: managed
```

### Reproducible demo or steps to reproduce from a blank project

NA

**Comment by brentvatne** [comment]
if you haven't opted in to using the new beta expo go client then it woulnd't be related

**Comment by dereknelson** [comment]
@brentvatne i assumed the "update available" was related to an remote endpoint that added a new SDK version but i am also speaking entirely out of my ass ¯\_(ツ)_/¯

**Comment by jonsamp** [comment]
@dereknelson where are you seeing "update available"? 

I have an M1 MacBook Pro, and I just created a managed project. Then I deleted the existing Expo Go app on my simulator and ran `expo start --ios`. 

Here's a video of this on SDK 40:

https://user-images.githubusercontent.com/6455018/113208246-8f515880-923f-11eb-8729-1e6916e56b94.mov

And here's a video of this on SDK 41 (beta):

https://user-images.githubusercontent.com/6455018/113208271-97a99380-923f-11eb-9e07-fe72b379dbcb.mov

I also tried loading the SDK 41 app on my Expo Go app on my iPhone, and it worked. Im trying to replicate your issue. Am I missing what you are trying to do in these demos? Are there other details that would help me see this issue?
